docs/explanations/apt_messages.md

```diff
@@ -53,7 +53,7 @@ where we deliver those messages are:
 
   ```
   *Your Ubuntu Pro subscription has EXPIRED*
-  The following security updates require Ubuntu Pro with 'esm-infra' enabled:
+  Get more security updates through Ubuntu Pro with 'esm-infra' enabled
```

Comment:
The problem I see is that it only apply for apps. For infra, the `require` message is more accurate

Reply:
That is true, what about if we only changed that for `esm-apps` then ?

Reply:
Yep, that's what I think we should do

Reply:
@renanrodrigo updated